### PR TITLE
added ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by manu-chroma**
This template will automatically show up in write area when creating issues in this repository. 
I've picked this template from ``jinja2`` project and made necessary changes to make it suitable for our project.

### CODE REVIEW

**Comment by mr-c** [comment]
Jenkins, okay to test

**Comment by tetron** [comment]
This looks great!

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Issue tracker is a tool to address bugs in cwltool (reference implementation) itself.
+Please use the [Gitter channel](https://gitter.im/common-workflow-language/common-workflow-language) or [Biostars forum](https://www.biostars.org/)
```
Comment (mr-c) [inline .github/ISSUE_TEMPLATE.md]: Please mention that posting on biostars should include the `cwl` tag

Comment (manu-chroma) [inline .github/ISSUE_TEMPLATE.md]: > Don't forget to use cwl tag when posting on Biostars Forum. 

Does this work?

Comment (mr-c) [inline .github/ISSUE_TEMPLATE.md]: Sure

**Pull request merged**